src/components/link.js

```diff
@@ -15,6 +15,7 @@ module.exports.Component = registerComponent('link', {
     highlightedColor: {default: '#24CAFF', type: 'color'},
     href: {default: ''},
     image: {type: 'asset'},
+    previewDistance: {default: 15, min: 1},
```

Comment:
This seems hard to understand and puts the burden on the developer. What about adapting the distance automatically based on the scale of the entity?

Reply:
That sounds doable.

Reply:
Done.